x/mongo/driver/url/url.go

```diff
@@ -0,0 +1,1218 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
Pulling in a modified version of the `"net/url"` package is unnecessary because the already-exported `url.PathUnescape` behaves the same as the now-exported `Unescape` with mode `EncodeUserPassword`. Please remove the unnecessary `url.go`.